internal/tools/modformatter/modformatter.go

```diff
@@ -188,7 +193,7 @@ func main() {
 			if *formatFile {
 				writeNeeded = true
 				trimName := strings.Split(modname, "github.com/DataDog/datadog-agent/")[1]
-				trimPath := strings.Split(*path, "github.com/DataDog/datadog-agent/")[1]
+				trimPath := "." + strings.Split(*path, *repoPath)[1]
```

Comment:
```suggestion
				trimPath := "." + strings.TrimPrefix(*path, *repoPath)
```
As I guess it will always be called with the current working directory path?